rero_ils/query.py

```diff
@@ -162,9 +99,8 @@ def documents_search_factory(self, search, query_parser=None):
 def viewcode_patron_search_factory(self, search, query_parser=None):
     """Search factory with viewcode or current patron."""
     search, urlkwargs = search_factory(self, search)
-    view = request.args.get('view')
     # Public interface
-    if view:
+    if view := request.args.get('view'):
```

Comment:
Two `if` can be an AND.

Reply:
same remarks: `:=` operator couldn't be use with other condition.

Reply:
in this case, I think two if seems more readable